### ISSUE TITLE
README.md not updated with releases

### ISSUE THREAD

**Issue by KevinRemhof** [ratgdo/homekit-ratgdo#108]
Suggestion: if you make changes in a release such as adding a login, please update the README.md file with the information. The release notes for v0.11.0 do specify the default username and password which is necessary. The README.md does not have this info yet.

And yes, I know I could put in a PR to address this. :)

**Comment by Damien514** [comment]
I tried "admin / password" and it worked. Clicking the setting icon, I was able to update to my own password.

**Comment by KevinRemhof** [comment]
Yes, admin/password works. My issue is that the README.md should have been updated to specify this. You had to read the release notes to find that.

My suggestion is an improvement to the overall process when creating a new release.

**Comment by jgstroud** [comment]
Yeah, this occurred to me right after I merged the PR that we should have included a README update with it.

**Comment by jgstroud** [comment]
LoL.... Then shortly after that I remembered that @donavanbecker loves updating the readme file and anticipated a fix by the next morning.

**Comment by donavanbecker** [comment]
> LoL.... Then shortly after that I remembered that @donavanbecker loves updating the readme file and anticipated a fix by the next morning.

Done: [#115](https://github.com/ratgdo/homekit-ratgdo/pull/115)
